languages/csharp/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -0,0 +1,57 @@
+In this exercise, you're an avid bird watcher that keeps track of how many birds have visited your garden in the last seven days.
+
+You have five tasks, all dealing with the numbers of birds that visited your garden.
+
+### 1. Calculate the total number of visiting birds
+
+Implement the `BirdCount.Total()` method to return the total number of birds that have visited your garden:
+
+```csharp
+int[] dayCounts = { 2, 5, 1, 4, 7, 0 };
+var birdCount = new BirdCount(dayCounts);
+birdCount.Total();
+// => 19
+```
+
+### 2. Calculate the number of busy days
+
+Some days are busier that others. A busy day is one where five or more birds have visited your garden.
+Implement the `BirdCount.BusyDays()` method to return the number of busy days:
+
+```csharp
+int[] dayCounts = { 2, 5, 1, 4, 7, 0 };
+var birdCount = new BirdCount(dayCounts);
+birdCount.BusyDays();
+// => 2
+```
+
+### 3. Check how many birds visited yesterday
+
+Implement the `BirdCount.Yesterday()` method to return how many birds visited your garden yesterday, which is the second last element:
+
+```csharp
+int[] dayCounts = { 2, 4, 1, 6, 7, 0 };
+var birdCount = new BirdCount(dayCounts);
+birdCount.Yesterday();
+// => 7
+```
+
+### 4. Check for a specific count
+
+Your neighbor is also a bird enthusiast. Occasionally, your neighbor wants to know if your garden was also visited by a specific number of birds. Implement the `BirdCount.Exact()` method that the number of birds to check as a parameter and returns a boolean that indicates if that exact number of birds has visited your garden:
```

Comment:
```suggestion
Your neighbor is also a bird enthusiast. Occasionally, she wants to know if your garden was also visited by a specific number of birds. Implement the `BirdCount.Exact()` method that the number of birds to check as a parameter and returns a boolean that indicates if that exact number of birds has visited your garden:
```

Reply:
I don't understand this one :) It feels like it should take two params - one for the day and one for the count on that day.

Reply:
Ah yes, you are right. That does make more sense. I'll add it.

Reply:
Well that would not work for what I'm trying to use the method for: getting students to use `Array.IndexOf`. Let me see if I can rephrase.

Reply:
So my goal is to have the student know about the `Array` class and its static methods: https://docs.microsoft.com/en-us/dotnet/api/system.array?view=netframework-4.8#methods Unfortunately, most of those methods are either complex or require complex parameters (predicates), which I don't want to use.